source/k64f_emac.c

```diff
@@ -583,7 +583,7 @@ static err_t k64f_low_level_output(struct netif *netif, struct pbuf *p)
       break;
   if (q != NULL) {
     // Allocate properly aligned buffer
-    psend = (uint8_t*)malloc(p->tot_len);
+    psend = (uint8_t*)pbuf_alloc(PBUF_RAW,p->tot_len,PBUF_POOL);
```

Comment:
1. What kind of alignment does `pbuf_alloc` guarantee? On K64F, the TX buffers must be aligned to 8 bytes.
2. Do we know for sure that `pbuf_alloc` is re-entrant?